packages/gatsby/src/utils/start-server.ts

```diff
@@ -92,20 +92,21 @@ export async function startServer(
     !isCI()
   ) {
     cancelDevJSNotice = showExperimentNoticeAfterTimeout(
-      `PRESERVE_WEBPACK_CACHE`,
-      report.stripIndent(`
-Your friendly Gatsby maintainers detected your site has more JavaScript than most sites! We're working to make your site's JS compile as quickly as possible by avoiding clearing your webpack cache as often.
+      `Don't clear webpack's cache as often`,
+      `We noticed your site has more JavaScript than most sites. We're working to make
+compiling your code faster by not clearing webpack's cache unless really
+necessary. This will make your local development server load faster.
```

Comment:
I would add a `at the risk of ...`

Reply:
we're inviting user testing because we're not sure there is a risk but we want to see what happens in the wild (but w/ controlled & informed testers). Doing a general release could be possible but is riskier as then everyone might see unresolved (and unknown) bugs.

Reply:
:shrug: ok